### DISCUSSION TITLE
<(is this the right categorie?) I’m new at coding, no experience and I’m trying to run this program I found on here but can’t, idk if I can do it in here or have to use a app>

### DISCUSSION THREAD

**Discussion by Winter-classisdisguised** [community/community#123284] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

If anyone can help that would be lovely

**Comment by ebndev** [comment]
Hi @Winter-classisdisguised, we're happy you're here! 

You are more likely to get a useful response if give a few more details about what program you're trying to run / what issues you're running into. That context might help someone give you a nudge in the right direction.
